docs/watchlists-feeds-reports.rst

```diff
@@ -0,0 +1,244 @@
+either alerts or hits, respectively; if both are ``False``, the Watchlist is effectively disabled.
+
+If you already have an existing Watchlist you wish to enhance, you can add Reports to the existing Watchlist.
+
+Enabling Alerting on a Watchlist
+++++++++++++++++++++++++++++++++
+Once you have the Watchlist configured with the IOCs that are generating the kinds of hits (results) you are after,
+you can enable Alerting for the Watchlist.
+
+::
+
+    >>> watchlist.enable_alerts()
+
+A Closer Look at IOCs
+---------------------
+In this document, we will only discuss the "v2" IOCs; other types of IOCs are only provided for backwards compatibility
+reasons, and are converted, internally, to this type.
+
+IOCs can be classified into two general types, depending on their ``match_type`` value:
+
+*Query IOCs* are those with a ``match_type`` of ``query``; their ``values_list`` contains a single string that
+specifies a query compatible with process searches.  For example, the following IOC looks for the process ``git.exe``
+that does *not* connect to one of a specified list of IP addresses:
+
+::
+
+    {
+        "id": "example_1",
+        "match_type": "query",
+        "values": ["process_name:git.exe NOT (netconn_ipv4:35.158.151.206 OR netconn_ipv4:1.1.244.78
+                    OR netconn_ipv4:80.18.61.229 OR netconn_ipv4:80.18.61.228)"]
+    }
+
+Query IOCs are searched every 5 minutes by the Carbon Black Cloud, and are tested against a rolling window of the
+last hour's worth of data for the organization.  (They will *not* generate hits or alerts for process attributes that
+were reported more than an hour in the past.)  They may employ any searchable field, and may employ complex query
+logic.
+
+*Ingress IOCs* are those with a ``match_type`` of ``equality`` or ``regex``; they use the ``field`` element to specify
+the name of a field to examine the value of, and the ``values_list`` element to specify a list of values to match
+against (in the case of ``match_type`` being ``equality``) or regular expressions to match against (in the case of
+``match_type`` being ``regex``).  For example, this IOC will match any process that initiates a connection to one of
+two listed IP addresses:
+
+::
+
+    {
+        "id": "example_2",
+        "match_type": "equality",
+        "field": "netconn_ipv4",
+        "values": ["8.8.8.8", "1.160.120.15"]
+    }
+
+This IOC will match any process running with an executable name beginning with "quake":
+
+::
+
+    {
+        "id": "example_3",
+        "match_type": "regex",
+        "field": "process_name",
+        "values": ["quake.*\\.exe"]
+    }
+
+(Note the use of the backslash to escape the '.' that separates the file extension from the name.  It must be doubled
+to escape it in Python itself.)
+
+Ingress IOCs are searched as soon as the data is received from any endpoint, and may use any documented process field
```

Comment:
Link out to the process fields

Reply:
Where can I find those?

Reply:
This is the only page we've got for that: https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/platform-search-fields/ But it's very confusing; how am I to know which is a "process" field?

Reply:
The fields marked with `PROCESS`

Reply:
Added a link in.